.gitignore

```diff
@@ -2,3 +2,6 @@
 node_modules
 install-*.sh
 install-*.bat
+install-*.ps1
+.install.sh.swp
```

Comment:
What we want to match here is vim swap files, which would be in the format of `.*.sw*`

Reply:
Okay I will test what's the generated files are and accordingly make changes